posthog/exceptions.py

```diff
@@ -43,6 +46,8 @@ def exception_reporting(exception: Exception, context: ExceptionContext) -> None
     Determines which exceptions to report and sends them to Sentry.
     Used through drf-exceptions-hog
     """
+    logger.exception(exception, path=context["request"].path)
```

Comment:
Sorry for all the iterations: Should we log for APIException's - these are things like validation failed, auth failed, throttling, etc.

https://www.django-rest-framework.org/api-guide/exceptions/

Reply:
Fair point, makes sense! Will log only those going to sentry.